flax/nnx/transforms/autodiff.py

```diff
@@ -233,14 +233,19 @@ def grad(
   tp.Callable[..., tp.Any]
   | tp.Callable[[tp.Callable[..., tp.Any]], tp.Callable[..., tp.Any]]
 ):
-  """Lifted version of ``jax.grad`` that can handle Modules / graph nodes as
-  arguments.
+  """A "lifted" version of `jax.grad <https://jax.readthedocs.io/en/latest/_autosummary/jax.grad.html#jax.grad>`_
```

Comment:
I know this was there but we can improve this while we are at it... can you remove this "lifted" terminology. I think "reference-aware" or something like that is better.

Reply:
Yes! SGTM. We have discussed not using "lifted" with you and team (cc @levskaya) before.